packages/ui-informed/src/controls/InputControl.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { type ComponentProps } from 'react';
+
+import { Input } from '../../../ui/src/components/input/Input';
```

Comment:
you should install `@utima/ui` as dependency in `package.json`. npm should automatically symlink it with current version in monorepository. After that you can use imports from `@utima/ui`

Reply:
I have some local troubles, it's fixed.